package/setup.py

```diff
@@ -662,6 +662,9 @@ def long_description(readme):
                                    # reduction functionality in encore
                   'tidynamics>=1.0.0', # For MSD analysis method
               ],
+              'dask': [
```

Comment:
This sounds to me like it should go under analysis? It's a "parallel analysis" thing after all?

Reply:
I think it should be separate from analysis — people might want analysis but without parallel things. Perhaps we can rename it to “parallel” as a more general thing?

Reply:
We had this discussion over in Discord: 

https://discord.com/channels/807348386012987462/1105792867143798835/1119002894008590337

Happy with either `dask` or `parallel`, but I agree it's better to have it separate.

[Note: With GitHub Discussion it would be easier to link such discussion. =)]

Reply:
I'd go with parallel then - dask is too unintuitive a key.

Reply:
Also please update pyproject.toml

Reply:
The current implementation uses the following notation: the "backend" variable is essentially the package name that you need to import. While I do agree that installing it is slightly unintuitive, I believe that documentation will fix that. Also, there is an explicit `available_backends` variable attached to each class, that helps people to choose the backend.

Reply:
I'm not sure how your comment relates to the discussion above. Could you please elaborate? I think the consensus here is to name this `parallel` and add both `dask` and `dask.distributed`.